docs/js/doc-page.js

```diff
@@ -60,6 +188,19 @@ export default ({ data, location }) => {
         {metaDescription && (
           <meta name="description" content={metaDescription} />
         )}
+        <script type="application/ld+json">
+          {JSON.stringify(
+            getDocPageSchema({
+              isLatest: slug.includes('docs/latest'),
+              slug,
+              title,
+              version: versions.version,
```

Comment:
This will set the schema's `version` to the version of Ambassador, right? That wouldn't be quite right, would it; the version there should be the web-page's version, not the Ambassador release it describes, right?